packages/UI/src/components/organisms/PositionCard/PositionCard.tsx

```diff
@@ -0,0 +1,178 @@
+import { useState, useRef, useLayoutEffect } from 'react';
+import Share from '../../../assets/icons/Link';
+import bookmark from '../../../assets/icons/rawSvg/bookmark.svg';
+import heart from '../../../assets/icons/rawSvg/heart.svg';
+import Box from '../../atoms/Box';
+import Button from '../../atoms/Button';
+import Tags from '../../atoms/Tags';
+import Typography from '../../atoms/Typography';
+import { Container, Actions, Details } from './Styled.PositionCard';
+import type { PositionCardProps } from '.';
+
+const content = {
+  title: 'Web Developer',
+  level: 'junior',
+  timeCommittment: '5 hrs per week',
+  role: ' We are looking for a junior web developer who is willing and eagerto learn various web technologies mainly in the field of React while working within a team of ambitious developers! Our goal is to designa platform that brings people of from a diverse range of specialties together to collaboratively design the best possible software solutions. This project provides a space for people to establish and develop their ideas into working projects with input from others.',
+  tags: ['unity', 'react', 'social', 'agile', 'digital media'],
+  benefits:
+    " Be proficient in React.js and/or Next.js (strongly prefer both or atleast Next) Experience with GitHub and version control using Git Backend experience preferred, especially using a CMS Be able tocommit at least 5 hours a week Self-sufficient learner Communicate with us on a weekly basis to see the progress you've been making",
+  expecations:
+    "  Be proficient in React.js and/or Next.js (strongly prefer both or at least Next) Experience with GitHub and version control using Git Backend experience preferred, especially using a CMS Be able to commit at least 5 hours a week Self-sufficient learner Communicate with us on a weekly basis to see the progress you've been making",
+};
```

Comment:
@Nouri-Anouar 
This should move to the stories file, it's content and content should be dynamic and not used in the React component file.

Reply:
Got it! makes sense!